readme.md

```diff
@@ -386,7 +386,7 @@
 - [Godot](https://github.com/Calinou/awesome-godot#readme) - Game engine.
 - [Open Source Games](https://github.com/leereilly/games#readme)
 - [Unity](https://github.com/RyanNielson/awesome-unity#readme) - Game engine.
-- [Chess](https://github.com/hkirat/awesome-chess#readme)
+- [Chess](https://github.com/mbiesiad/awesome-chess#readme)
```

Comment:
Since this is a new list, you need to include a description of what Chess is.

Reply:
Done, added.